std/node/tests/node_modules/left-pad/index.js

```diff
@@ -16,7 +16,7 @@ var cache = [
   "      ",
   "       ",
   "        ",
-  "         "
+  "         ",
```

Comment:
This seems totally unrelated to the PR.

Reply:
I think this is from when I launched `deno fmt` globally by accident. Oops